internal/backup/volumesnapshotcontent_action.go

```diff
@@ -122,43 +101,6 @@ func (p *VolumeSnapshotContentBackupItemAction) Progress(operationID string, bac
 		return progress, biav2.InvalidOperationIDError(operationID)
 	}
 
-	// The operationId is of the form <volumesnapshotcontent-name>/<started-time>
-	operationsIDParts := strings.Split(operationID, "/")
```

Comment:
The chunk to check if `operationID == ""` seems can also be removed?

Reply:
Modified.